WcaOnRails/app/views/admin/complete_persons.html.erb

```diff
@@ -114,7 +114,8 @@
       <% end %>
       </tbody>
     <% end %>
-    <%= f.input :continue_batch, as: :boolean, label: "Continue with the next batch of newcomers if there are more pending entries", hint: "If you are redirected to the 'Create Newcomers' entry point, it means that there are no more pending entries.", input_html: { checked: competition_ids.present? } %>
+    <%= f.input :continue_batch, as: :boolean, label: "Continue with the next batch of newcomers if there are more pending entries", input_html: { checked: competition_ids.present? } %>
+    <%= f.input :clear_inbox_persons_after_finish, as: :boolean, label: "Clear InboxPersons if there are no more pending entries", hint: "If you are redirected to the 'Create Newcomers' entry point, it means that there are no more pending entries.", input_html: { checked: competition_ids.present? } %>
```

Comment:
You moved the hint from `continue_batch` to `clear_inbox_persons_after_finish`. This could lead to confusion regarding what the hint is actually referring to. Also, every `f.input` needs an explicit mention of `hint` unless it's not in the i18n YAMLs (and for results posting, we don't want that!) so you have to add something like `hint: false` when you don't want any hint